app/rest/connector-generator/pom.xml

```diff
@@ -99,6 +99,16 @@
       <artifactId>value</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>com.github.spullara.mustache.java</groupId>
+      <artifactId>compiler</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
```

Comment:
Wonder can we do it without this extra dependency? Seems overkill for just escaping the SVG XML?

Reply:
A simple JS version that could be copied over perhaps? https://github.com/tigt/mini-svg-data-uri/blob/master/index.js

Reply:
I would not mind replacing that with something (that I did not code) but that won't remove the guava dependency from the backend, we have it in `connector-generator` and Swagger brings it in also

Reply:
Using SVG for icons is perfectly fine and we can embed those as regular `<img>` elements and even apply transformations thru custom pipes if necessary.

If file format is an issue (which it shouldn't), a workaround is to grab whatever the backend returns (either a SVG, a PNG, a TIFF, a GIF, whatever), open a blank canvas object, embed the bitmap there (resize it if necessary), and then use the Canvas Web API to translate the canvas representation into binary data (Base64 encoded) and inject it as a PNG generated in memory into any `<img>` tag. We can implement a custom a NG pipe so we can reuse this same logic globally, same as we did with that path from the connector icons.

Doing the above is trivial and just takes 30 minutes of fiddling with TypeScript code, but if the backend can return a path to a SVG file, we can get away with that.

Reply:
@jimmidyson given that we already have guava dependency in our fat jar (brought in by Swagger) can we move forward with this without removing guava?